src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -208,7 +208,7 @@ public function getRouteCollectionForRequest(Request $request)
                 return $collection;
             } else {
                 if ($document instanceof ExtensionBehavior) {
-                    $documentSegments = $document->getExtensionsData()['excerpt']['segments'];
```

Comment:
please adjust this in the `Sulu\Bundle\RouteBundle\Routing\RouteProvider` too

Reply:
@nnatter did adjust it.